CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## Improvements
 
 ## Bug Fixes
+- PR #3279: Correct pure virtual declaration in manifold_inputs_t
```

Comment:
@wphicks can you remove the changelog entry?

Reply:
I'd be happy to, but based on Slack discussion, this should be unnecessary. The changelog will get correctly updated/replaced by the new system.